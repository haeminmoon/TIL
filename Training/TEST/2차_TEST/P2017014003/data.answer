Case #1
11
Case #2
23
Case #3
59
Case #4
28
Case #5
7
Case #6
25
Case #7
29
Case #8
66
Case #9
43
Case #10
0
Case #11
0
Case #12
23
Case #13
24
Case #14
79
Case #15
5
Case #16
61
Case #17
50
Case #18
2
Case #19
25
Case #20
0
Case #21
20
Case #22
60
Case #23
30
Case #24
12
Case #25
49
Case #26
33
Case #27
28
Case #28
25
Case #29
18
Case #30
1
Case #31
16
Case #32
29
Case #33
52
Case #34
5
Case #35
23
Case #36
5
Case #37
11
Case #38
37
Case #39
13
Case #40
26
Case #41
19
Case #42
10
Case #43
27
Case #44
0
Case #45
26
Case #46
0
Case #47
28
Case #48
4
Case #49
11
Case #50
18
Case #51
18
Case #52
18
Case #53
38
Case #54
25
Case #55
34
Case #56
16
Case #57
18
Case #58
28
Case #59
9
Case #60
7
Case #61
25
Case #62
0
Case #63
38
Case #64
46
Case #65
41
Case #66
27
Case #67
4
Case #68
21
Case #69
4
Case #70
25
Case #71
0
Case #72
94
Case #73
42
Case #74
58
Case #75
31
Case #76
62
Case #77
38
Case #78
17
Case #79
23
Case #80
34
Case #81
0
Case #82
30
Case #83
0
Case #84
0
Case #85
1
Case #86
30
Case #87
38
Case #88
15
Case #89
35
Case #90
39
Case #91
35
Case #92
79
Case #93
41
Case #94
3
Case #95
25
Case #96
0
Case #97
57
Case #98
20
Case #99
33
Case #100
50
