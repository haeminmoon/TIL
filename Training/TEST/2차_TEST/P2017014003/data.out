Case #1
59
Case #2
10
Case #3
6
Case #4
35
Case #5
45
Case #6
28
Case #7
39
Case #8
71
Case #9
0
Case #10
23
Case #11
35
Case #12
81
Case #13
20
Case #14
13
Case #15
58
Case #16
17
Case #17
42
Case #18
37
Case #19
44
Case #20
18
Case #21
70
Case #22
15
Case #23
10
Case #24
58
Case #25
9
Case #26
25
Case #27
29
Case #28
52
Case #29
26
Case #30
0
Case #31
33
Case #32
18
Case #33
62
Case #34
49
Case #35
36
Case #36
14
Case #37
27
Case #38
5
Case #39
19
Case #40
2
Case #41
26
Case #42
11
Case #43
21
Case #44
0
Case #45
18
Case #46
0
Case #47
55
Case #48
0
Case #49
42
Case #50
29
Case #51
13
Case #52
37
Case #53
36
Case #54
35
Case #55
11
Case #56
37
Case #57
26
Case #58
78
Case #59
24
Case #60
30
Case #61
5
Case #62
0
Case #63
25
Case #64
34
Case #65
37
Case #66
46
Case #67
0
Case #68
6
Case #69
3
Case #70
31
Case #71
3
Case #72
0
Case #73
7
Case #74
22
Case #75
0
Case #76
34
Case #77
2
Case #78
0
Case #79
24
Case #80
38
Case #81
20
Case #82
51
Case #83
27
Case #84
32
Case #85
49
Case #86
9
Case #87
26
Case #88
56
Case #89
7
Case #90
0
Case #91
27
Case #92
17
Case #93
0
Case #94
32
Case #95
39
Case #96
27
Case #97
52
Case #98
51
Case #99
5
Case #100
53
